scripts/validate-gentxs.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+DAEMON_HOME="/tmp/simd$(date +%s)"
+RANDOM_KEY="randomvalidatorkey"
+
+echo "#############################################"
+echo "### Ensure to set the below ENV settings ###"
+echo "#############################################"
+echo "
+DAEMON= # ex: simd
+CHAIN_ID= # ex: testnet-1
+DENOM= # ex: ustake
+GH_URL= # ex: https://github.com/cosmos/cosmos-sdk
+BINARY_VERSION= # ex :v0.43.0-beta1
+GO_VERSION=1.15.2
+PRELAUNCH_GENESIS_URL= # ex: https://raw.githubusercontent.com/cosmos/cosmos-sdk/master/$CHAIN_ID/genesis-prelaunch.json
+GENTXS_DIR= # ex: $GOPATH/github.com/cosmos/mainnet/$CHAIN_ID/gentxs"
+echo
+
+if [[ -z "${GH_URL}" ]]; then
+  echo "GH_URL in not set, required. Ex: https://github.com/cosmos/cosmos-sdk"
+  exit 0
+fi
+if [[ -z "${DAEMON}" ]]; then
+  echo "DAEMON is not set, required. Ex: simd, gaiad etc"
+  exit 0
+fi
+if [[ -z "${DENOM}" ]]; then
+  echo "DENOM in not set, required. Ex: stake, uatom etc"
+  exit 0
+fi
+if [[ -z "${GO_VERSION}" ]]; then
+  echo "GO_VERSION in not set, required. Ex: 1.15.2, 1.16.6 etc."
+  exit 0
+fi
+if [[ -z "${CHAIN_ID}" ]]; then
+  echo "CHAIN_ID in not set, required."
+  exit 0
+fi
+if [[ -z "${PRELAUNCH_GENESIS_URL}" ]]; then
+  echo "PRELAUNCH_GENESIS_URL (genesis file url) in not set, required."
+  exit 0
+fi
+if [[ -z "${GENTXS_DIR}" ]]; then
+  echo "GENTXS_DIR in not set, required."
+  exit 0
+fi
+
+command_exists () {
+    type "$1" &> /dev/null ;
+}
+
+if command_exists go ; then
+    echo "Golang is already installed"
+else
+  echo "Install dependencies"
```

Comment:
I think it would be better to make this optional, as many distros exist that don't use `apt`. Perhaps prompt the user and let them know it will be assuming `apt`. If they say no, just exit and prompt them to install `go` manually?